front/lib/api/assistant/configuration.ts

```diff
@@ -164,7 +195,38 @@ export async function updateAgentConfiguration(
   const agentConfig = await AgentConfiguration.findOne({
```

Comment:
I would retrieve only AgentConfiguration, update it and then call `getAgentConciguration` to return?

Reply:
Done!